src/encoded/submission_status.py

```diff
@@ -133,12 +145,17 @@ def process_files_metadata(files_metadata):
     return {
         "is_upload_complete": is_upload_complete,
         "num_submitted_files": len(submitted_files),
+        "num_fileset_files": len(files_metadata),
         "date_uploaded": date_uploaded,
         "file_formats": ", ".join(file_formats),
         "num_files_copied_to_o2": num_files_copied_to_o2,
     }
 
 
+def is_accession(s: str) -> bool:
+    return len(s) == 12 and s.startswith("SMA")
```

Comment:
This will work but there is another function that will do this for you definitively: `schema_formats.is_accession_for_server`